modules/virt-querying-the-node-exporter-service-for-metrics.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-querying-the-node-exporter-service-for-metrics-_{context}"]
+= Querying the node-exporter service for metrics
+
+Metrics are exposed for virtual machines through an HTTP service endpoint under the `/metrics` canonical name. When you query for metrics, Prometheus directly scrapes the metrics from the metrics endpoint exposed by the virtual machines and presents these metrics for viewing.
```

Comment:
Not sure why the text is italicized in this module. It is rendering fine though.

Reply:
That is weird! I removed the underscores I added to the bracketed text for user supplied values. Let's hope that will fix the issue and remove the unwanted italics.